src/components/CommentForm/CommentForm.tsx

```diff
@@ -387,7 +391,7 @@ export const CommentForm = ({
         <div className={bottomContainer}>
           <div className={buttonContainerStyles}>
             <Button type="submit" size="small">
-              Post your comment
+              <div className={buttonTextStyles}>Post your comment</div>
```

Comment:
do we need to use a div? could the styles not be applied to the button?

Reply:
You can [override styles](https://www.theguardian.design/2a1e5182b/p/2619e3-overriding-styles) of Source components. It would be useful to understand why we need to in this case... is it just for visual parity, or is there a presentational reason?

Reply:
It's for presentational reasons. The button text is larger than the standard text size used throughput discussion (textSans.small) and seemed a bit, well, big. But I'm happy to take this out and stick with src. I don't have strong opinions here.

## Before
![Screenshot 2020-03-21 at 22 47 15](https://user-images.githubusercontent.com/1336821/77238224-57dcb480-6bc6-11ea-8454-c8a8c090200a.jpg)

## After
![Screenshot 2020-03-21 at 22 47 02](https://user-images.githubusercontent.com/1336821/77238221-54e1c400-6bc6-11ea-817a-07d7a3bdd668.jpg)

Reply:
Re: passing classNames. It's not my favourite thing to do because it abstracts the styling in a way that doesn't scale and creates this strange coupling where I lose control over what styles might win in a war. If I wrap it in a div then I control things. But it's an extra div 🤷‍♂

Reply:
Right on. I'll discuss the text size thing with Akemi. It's certainly something we should keep an eye on.

Regarding the extra div, I don't mind as long as it still gets read out by screenreaders 🙂

Reply:
> as long as it still gets read out by screenreaders

Good call. I tested using a screenreader and it works fine 👍